### PR TITLE
Async img loading

### PR DESCRIPTION
**Pull request by noahadams**

### CODE REVIEW

File: www/v2/docs/index.md [hunk 1]
```diff
@@ -60,17 +60,32 @@ any element that loads external resources!**):
...
-      Mobify.ResizeImages.resize(images);
+        var intervalId = setInterval(function(){
+            if (window.Mobify) {
+                var images = document.querySelectorAll('img[x-src]');
+                if (images.length > 0) {
+                    Mobify.ResizeImages.resize( images, {
+                        maxWidth: 320   
+                    });
+                }
+                // When the document has finished loading, stop loading images.
```
Comment (noahadams) [inline www/v2/docs/index.md]: stop checking for new images

File: examples/resizeImages-no-capturing/index.html [hunk 2]
```diff
@@ -12,20 +12,28 @@
...
+<script>
+    var intervalId = setInterval(function(){
+        if (window.Mobify) {
+            var images = document.querySelectorAll('img[x-src]');
+            if (images.length > 0) {
+                Mobify.ResizeImages.resize( images, {
+                    maxWidth: 320   
+                });
+            }
+            // When the document has finished loading, stop loading images.
```
Comment (noahadams) [inline examples/resizeImages-no-capturing/index.html]: stop checking for new images

**Comment by noahadams** [comment]
@jansepar I made a couple of comments about comment changes I'd like to see, once those are made you have my thumbs up

**Comment by jansepar** [comment]
Btw thanks @johnboxall for the idea to do this async!

**Comment by jansepar** [comment]
@noahadams comment addressed. Thanks for having a look at this, and for your :+1: !

**Pull request merged**